.github/workflows/go-tests.yaml

```diff
@@ -17,7 +17,21 @@ permissions:
 jobs:
   test:
     runs-on: mal-ubuntu-latest-8-core
-    container: cgr.dev/chainguard/wolfi-base:latest
+    container:
+      image: cgr.dev/chainguard/wolfi-base@sha256:eeb70e74e2ac07d3c80a30150bf473970c8b51a57f06daef3e4d065ac52489bc
```

Comment:
I wanted to wait until CI was running consistently before locking everything down.